ee/localserver/server.go

```diff
@@ -35,6 +35,7 @@ var portList = []int{
 	22322,
 }
 
+//go:generate mockery --name Querier
```

Comment:
Could also put this into the `_test.go` file. You're call.

I'd add the comment about where mockery is from to this.